recirq/quantum_chess/dynamic_look_ahead_heuristic_circuit_transformer_test.py

```diff
@@ -0,0 +1,330 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+import pytest
+from collections import deque
+
+import cirq
+
+import recirq.quantum_chess.circuit_transformer as ct
+
+a0 = cirq.NamedQubit('a0')
+a1 = cirq.NamedQubit('a1')
+a2 = cirq.NamedQubit('a2')
+a3 = cirq.NamedQubit('a3')
+a4 = cirq.NamedQubit('a4')
+a5 = cirq.NamedQubit('a5')
+a6 = cirq.NamedQubit('a6')
+a7 = cirq.NamedQubit('a7')
```

Comment:
Optional:  I think the style guide would have these  e capitalized (i.e. A7), but this goes against chess algebraic style, so I am not sure which is better.

Reply:
i'm leaning towards using lower case for consistency since that's currently used in https://github.com/quantumlib/ReCirq/blob/master/recirq/quantum_chess/constants.py and https://github.com/quantumlib/ReCirq/blob/master/recirq/quantum_chess/circuit_transformer_test.py.

Reply:
I'd agree on lower, for consistency with chess.